code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -48,34 +48,32 @@
 		if(14)
 			new /obj/item/nullrod/scythe/talking(src)
 		if(15)
-			new /obj/item/nullrod/armblade(src)
```

Comment:
Again. Please actually fix this issue instead of removing it, especially since *you* are the one who caused it.

Reply:
All you would have to do is remove the abstract flag from the item. It isn't abstract, it is very real.